### PR TITLE
[ORBIT] Add Statistics Section

### PR DESCRIPTION
**Pull request by CharlyMartin**
This PR adds the cards in the Statistics section of the orbit page.

---

CleanShot 2024-06-27 at 15 19 34

[Notion](https://www.notion.so/filecoin/FF-V4-ORBIT-Create-Statistics-Card-0bb8fb50f6ab48a296c15389f06eabd3?pvs=4)

### CODE REVIEW

**Comment by barbaraperic** [comment]
Safari desktop v.16

Screenshot 2024-07-02 at 15 16 10

**Comment by barbaraperic** [comment]
In the designs, max width of the image is `114px`

Screenshot 2024-07-02 at 15 25 31

File: src/app/_components/StatisticCard.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      <figure className="grid w-1/3 place-items-center rounded bg-brand-500">
+        <Icon component={icon} color="brand-200" size={40} />
+      </figure>
+
+      <div className="flex w-2/3 place-items-center p-4">
+        <div className="space-y-2">
+          <span className="text-4xl font-light text-brand-100">
+            {value.toLocaleString()}+
+          </span>
+          <p className="text-sm leading-4 text-brand-200">{description}</p>
```
Comment (barbaraperic) [inline src/app/_components/StatisticCard.tsx:25]: in the designs, font size is 12px?

**Review by barbaraperic — APPROVED** [review]
Left some comments, besides some design stuff, good to go 👌

File: src/app/_components/StatisticCard.tsx (hunk 1 above)
Comment (mirhamasala) [reply src/app/_components/StatisticCard.tsx:25]: It's very small. I'll talk to Filipa! Thank you.

**Pull request merged**